src/components/avatar/AvatarBase.tsx

```diff
@@ -103,21 +58,22 @@ export const AvatarBase = ({
   className,
   shape,
   size,
+  style,
 }: AvatarBaseProps) => (
-  <Wrapper
-    $bordered={bordered}
-    $shape={shape}
-    $size={size}
-    className={className}
+  <div
+    className={clsx(styles.wrapper, className)}
+    style={{
+      ...style,
+      '--background-color': backgroundColor || `${theme.gray100}`,
+      '--background-image': `url(${backgroundUrl})`,
+      '--background-position': backgroundPosition || 'center',
+      '--background-size': backgroundSize || '',
+      '--border': bordered ? `${size / 16}px solid ${theme.gray0}` : '',
+      '--border-radius': avatarShapes[shape],
+      '--height': `${size}px`,
+      '--width': `${size}px`,
```

Comment:
Should this be namespaced like `--amino-avatar-base-background-color`?

Reply:
Yes it should. thanks